CultivateMVP/JournalOptionsScene/JournalOptionsViewModel.swift

```diff
@@ -1,17 +1,37 @@
 import Foundation
+import RxCocoa
+import RxSwift
 
 protocol JournalOptionsViewModelType {
-    var entries: [Entry] { get }
-    var moodEntry: MoodEntry { get }
+    var viewState: Driver<JournalOptionsViewState> { get }
 }
 
 struct JournalOptionsViewModel: JournalOptionsViewModelType {
```

Comment:
Maybe rename it until it's too late to what it really does? Like "JournalFilling" or "JournalQuestions"?